### DISCUSSION TITLE
Is there a self-hosted dashboard?

### DISCUSSION THREAD

**Discussion by csrn** [aspen-cloud/triplit#146] [Q&A] [answered]
Hi! 

Just a quick question: I have read that self-hosted triplit instances can be connected to your cloud's dashboard. This would be a show stopper for my use case where everything needs to be hosted on-premise. 

Did I miss something or is this cloud backend mandatory? Thank you!

**Comment by matlin** [comment, marked as answer]
Using the Triplit Dashboard is totally optional, you'll just need to manage more configuration. You can read this [page of the docs about deploying completely standalone](https://www.triplit.dev/docs/self-hosting).

**Reply by csrn** [reply]
Okay, thank you very much.
